test/functional/ui/wildmode_spec.lua

```diff
@@ -30,3 +31,34 @@ describe("'wildmode'", function()
     end)
   end)
 end)
+
+describe('command line completion', function()
+  local screen
+
+  before_each(function()
+    clear()
+    screen = Screen.new(40, 5)
+    screen:attach()
+    screen:set_default_attr_ignore({{bold=true, foreground=Screen.colors.Blue}})
+  end)
+
+  after_each(function()
+    os.remove('Xtest-functional-viml-compl-dir')
```

Comment:
I dislike helpers.rmdir for two reasons:

1. It deletes all files/directories in the directory you want to delete, but fails if a subdirectory is non-empty. This seems like an odd in-between case between the 'known' directory rmovers `rm -r` and `rmdir`.

1. It only works when the path separator is "/", which I'm explicitely trying to avoid as much as possible (note the `<bs>` after the `<tab>` in the test).

Also, I think it's valuable to generally use the "most specialized" function in tests, so it fails if there aren't the circumstances you expect. Of course, you don't want indeterministic behavior, but if really somehow a file or directory ends up in `Xtest-function-viml-compl-dir`, we probably want to know about it.